### PR TITLE
UML-1099: Updated content and translation

### PR DESCRIPTION
**Pull request by SeemaMenon**
# Purpose

https://opgtransform.atlassian.net/browse/UML-1099

Fixes UML-1099

## Approach

_Explain how your code addresses the purpose of the change_

## Learning

_Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the Use a Lasting Power of Attorney service_

## Checklist

* [X] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [X] I have added welsh translation tags and updated translation files
* [ ] The product team have tested these changes

### CODE REVIEW

File: service-front/app/languages/cy/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -4852,6 +4852,28 @@ msgctxt "error"
...
+msgid "Check your email"
+msgstr "Gwiriwch eich e-bost"
+
+#: src/Actor/templates/actor/create-account-success.html.twig:23
+msgid ""
+"The email should arrive in a few minutes.\n"
+"                        If it's not in your inbox, check your spam, bulk or "
+"junk email inbox.\n"
+"                        If you've not received the email, we can <a href="
+"\"%link%\" class=\"govuk-link\">resend the activation email</a>."
```
Comment (GemTay) [inline service-front/app/languages/cy/LC_MESSAGES/messages.po:4865]: I dont think this link looks quite right...

File: service-front/app/languages/messages.pot [hunk 2]
```diff
@@ -3359,3 +3359,18 @@ msgstr ""
...
 msgstr ""
+
+#: src/Actor/templates/actor/create-account-success.html.twig:19
+msgid "Check your email"
+msgstr ""
+
+#: src/Actor/templates/actor/create-account-success.html.twig:23
+msgid ""
+"The email should arrive in a few minutes.\n"
+"                        If it's not in your inbox, check your spam, bulk or junk email inbox.\n"
```
Comment (GemTay) [inline service-front/app/languages/messages.pot:3370]: Not sure why the translation tool has added all these spaces?

File: service-front/app/src/Actor/templates/actor/create-account-success.html.twig [hunk 3]
```diff
@@ -16,18 +16,19 @@
 
             <div class="govuk-grid-column-two-thirds">
 
-                <h1 class="govuk-heading-l">{% trans %}Activate your account{% endtrans %}</h1>
+                <h1 class="govuk-heading-l">{% trans %}Check your email{% endtrans %}</h1>
 
                 <p class="govuk-body">{% trans with {'%email%': emailAddress} %}We've emailed a link to %email%. Click on the link to activate your account. The link will work for the next 24 hours.{% endtrans %}</p>
```
Comment (GemTay) [inline service-front/app/src/Actor/templates/actor/create-account-success.html.twig:21]: This text in this paragraph needs to be slightly larger looking at the content image in the ticket, perhaps a "govuk-body-l"? Also the email address needs to be in bold

**Review by GemTay — CHANGES_REQUESTED** [review]
few tweaks

File: service-front/app/languages/messages.pot (hunk 2 above)
Comment (SeemaMenon) [reply service-front/app/languages/messages.pot]: not sure...have pulled from master and tried generating translations again.

File: service-front/app/languages/cy/LC_MESSAGES/messages.po (hunk 1 above)
Comment (SeemaMenon) [reply service-front/app/languages/cy/LC_MESSAGES/messages.po]: fixed looks like

**Review by cooperaj — APPROVED** [review]

**Review by GemTay — APPROVED** [review]
looks good :)

**Pull request merged**